FhirToDataLake/test/Microsoft.Health.Fhir.Synapse.Core.UnitTests/Jobs/DicomToDataLakeProcessingJobTests.cs

```diff
@@ -59,7 +59,7 @@ public async Task GivenValidDataClient_WhenExecute_ThenTheDataShouldBeSavedToBlo
             Assert.Equal(3, result.SearchCount["Dicom"]);
             Assert.Equal(3, result.ProcessedCount["Dicom"]);
             Assert.Equal(0, result.SkippedCount["Dicom"]);
-            Assert.Equal(911016, result.ProcessedDataSizeInTotal);
```

Comment:
Quan is fixing this in PR #249. I think we can keep the check here.

Reply:
Sure